packages/fxa-auth-server/lib/senders/templates/subscriptionPaymentFailed.html

```diff
@@ -0,0 +1,52 @@
+{{> subscriptionIcon}}
+
+<table class="featured-story featured-story--top" cellspacing="0" cellpadding="0">
+ <tr>
+  <td style="padding-bottom: 20px;">
+   <table cellspacing="0" cellpadding="0">
+    <tr>
+     <td class="featured-story__inner" style="background: #fff;">
+      <table cellspacing="0" cellpadding="0">
+       <tr>
+        <td class="featured-story__content-inner" style="padding: 32px 30px 45px;">
+         <table cellspacing="0" cellpadding="0">
+          <tr>
+           <td class="featured-story__heading featured-story--top__heading" style="background: #fff;" max-width="640" width="100%" align="left">
+            <table cellspacing="0" cellpadding="0">
+             <tr>
+              <td style="font-family:Helvetica, Geneva, Tahoma, Verdana, sans-serif; font-size: 22px; color: #464646;text-align:center !important;" width="640" align="center">
+                <b>{{t "Sorry, we're having trouble with your payment" }}</b>
+              </td>
+             </tr>
+            </table>
+           </td>
+          </tr>
+          <tr>
+           <td class="featured-story__copy" style="background: #fff;" width="640" align="center">
+            <table cellspacing="0" cellpadding="0">
+             <tr>
+              <td style="font-family:Helvetica, Geneva, Tahoma, Verdana, sans-serif; font-size: 16px; line-height: 22px; color: #555555; padding-top: 16px;text-align:left !important;" align="left">
```

Comment:
What determines class-based style versus inline?

Reply:
My answer: there is no determination, our emails need to be cleaned up. Lol. Someone correct me if I'm wrong. :sweat_smile:

Reply:
FWIW, this email HTML source comes out of the ZIP file exported from salesforce linked [from the overall epic](https://jira.mozilla.com/browse/FXA-1147). 

HTML email is enough of a dark art to me that I'm trying to keep from doing much besides letting my editor reformat the HTML - and I'm afraid that might be breaking things.

If we want to review / refine these templates overall, I think that would be a good follow-up